doc/commands/download.8.rst

```diff
@@ -44,6 +44,8 @@ Options
 ``--alldeps``
     | To be used together with ``--resolve``, it downloads all dependencies, not skipping the already installed ones.
 
+``--destdir=<path>``
+    Set directory to download packages to. Default location is the working directory
```

Comment:
Update of the description is still needed, see above: "used for downloading ... to the current working directory.". Also minor: add dot there at the end of the sentence :slightly_smiling_face:

Reply:
Another minor: maybe add a new example to the section below?

Reply:
Done